### PR TITLE
add `logLevel` support to `@next/bundle-analyzer`

### PR DESCRIPTION
**Pull request by quisido**
### What?

Allows `logLevel` for `BundleAnalyzerPlugin` to be configurable.

### Why?

Because the logs are super annoying, and I want to silence them, which is a feature of `BundleAnalyzerPlugin` already.

### How?

Passes `logLevel` to `BundleAnalyzerPlugin`.

### CODE REVIEW

**Comment by quisido** [comment]
@timneutkens @ijjk
 @shuding @huozhi @feedthejim @ztanner @wyattjoh

As code owners, can you share the process for getting a PR reviewed? The 400 open pull requests make this kind of intimidating, but this change is fairly trivial.

File: packages/next-bundle-analyzer/index.d.ts [hunk 1]
```diff
@@ -4,6 +4,12 @@ declare function NextBundleAnalyzer(options?: {
   enabled?: boolean
   openAnalyzer?: boolean
   analyzerMode?: 'json' | 'static'
+
+  /**
+   * Log level. Can be "info", "warn", "error" or "silent".
+   * @default 'info'
+   */
+  logLevel?: "info" | "warn" | "error" | "silent" | undefined
```
Comment (huozhi) [inline packages/next-bundle-analyzer/index.d.ts:12]: Let's align the quotes
```suggestion
  /**
   * Log level. Can be 'info', 'warn', 'error' or 'silent'.
   * @default 'info'
   */
  logLevel?: 'info' | 'warn' | 'error' | 'silent' | undefined
```

Comment (quisido) [reply packages/next-bundle-analyzer/index.d.ts]: Thank you! I've merged your change. I copy-pasted this from the bundleAnalyzer repo, so I didn't notice the style change until after I commit. 😔

**Comment by ijjk** [comment]
# Stats from current PR

<details open>
<summary><strong>Default Build</strong></summary>

<br/>

<details>
<summary><strong>General</strong></summary>

|  | vercel/next.js canary  | quisido/next.js patch-3 | Change |
| - | - | - | - |
| buildDuration | 10.4s | 10.5s | N/A |
| buildDurationCached | 6s | 5.8s | N/A |
| nodeModulesSize | 199 MB | 199 MB | ✓ |
| nextStartRea..uration (ms) | 435ms | 408ms | N/A |

</details>

<details>
<summary><strong>Client Bundles (main, webpack)</strong></summary>

|  | vercel/next.js canary  | quisido/next.js patch-3 | Change |
| - | - | - | - |
| 170-HASH.js gzip | 26.7 kB | 26.7 kB | N/A |
| 199.HASH.js gzip | 181 B | 185 B | N/A |
| 3f784ff6-HASH.js gzip | 53.3 kB | 53.3 kB | ✓ |
| framework-HASH.js gzip | 45.2 kB | 45.2 kB | ✓ |
| main-app-HASH.js gzip | 240 B | 241 B | N/A |
| main-HASH.js gzip | 31.7 kB | 31.7 kB | N/A |
| webpack-HASH.js gzip | 1.7 kB | 1.7 kB | N/A |
| Overall change | 98.5 kB | 98.5 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | quisido/next.js patch-3 | Change |
| - | - | - | - |
| polyfills-HASH.js gzip | 31 kB | 31 kB | ✓ |
| Overall change | 31 kB | 31 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | vercel/next.js canary  | quisido/next.js patch-3 | Change |
| - | - | - | - |
| _app-HASH.js gzip | 195 B | 194 B | N/A |
| _error-HASH.js gzip | 183 B | 182 B | N/A |
| amp-HASH.js gzip | 501 B | 501 B | ✓ |
| css-HASH.js gzip | 321 B | 321 B | ✓ |
| dynamic-HASH.js gzip | 2.5 kB | 2.5 kB | N/A |
| edge-ssr-HASH.js gzip | 255 B | 255 B | ✓ |
| head-HASH.js gzip | 349 B | 350 B | N/A |
| hooks-HASH.js gzip | 368 B | 369 B | N/A |
| image-HASH.js gzip | 4.27 kB | 4.27 kB | N/A |
| index-HASH.js gzip | 255 B | 256 B | N/A |
| link-HASH.js gzip | 2.6 kB | 2.6 kB | N/A |
| routerDirect..HASH.js gzip | 311 B | 309 B | N/A |
| script-HASH.js gzip | 384 B | 384 B | ✓ |
| withRouter-HASH.js gzip | 307 B | 306 B | N/A |
| 1afbb74e6ecf..834.css gzip | 106 B | 106 B | ✓ |
| Overall change | 1.57 kB | 1.57 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | quisido/next.js patch-3 | Change |
| - | - | - | - |
| _buildManifest.js gzip | 483 B | 483 B | ✓ |
| Overall change | 483 B | 483 B | ✓ |

</details>

<details>
<summary><strong>Rendered Page Sizes</strong></summary>

|  | vercel/next.js canary  | quisido/next.js patch-3 | Change |
| - | - | - | - |
| index.html gzip | 529 B | 529 B | ✓ |
| link.html gzip | 542 B | 542 B | ✓ |
| withRouter.html gzip | 525 B | 524 B | N/A |
| Overall change | 1.07 kB | 1.07 kB | ✓ |

</details>

<details>
<summary><strong>Edge SSR bundle Size</strong></summary>

|  | vercel/next.js canary  | quisido/next.js patch-3 | Change |
| - | - | - | - |
| edge-ssr.js gzip | 93.5 kB | 93.5 kB | N/A |
| page.js gzip | 146 kB | 146 kB | N/A |
| Overall change | 0 B | 0 B | ✓ |

</details>

<details>
<summary><strong>Middleware size</strong></summary>

|  | vercel/next.js canary  | quisido/next.js patch-3 | Change |
| - | - | - | - |
| middleware-b..fest.js gzip | 623 B | 624 B | N/A |
| middleware-r..fest.js gzip | 151 B | 151 B | ✓ |
| middleware.js gzip | 37.4 kB | 37.4 kB | N/A |
| edge-runtime..pack.js gzip | 1.92 kB | 1.92 kB | ✓ |
| Overall change | 2.07 kB | 2.07 kB | ✓ |

</details>

<details>
<summary><strong>Next Runtimes</strong></summary>

|  | vercel/next.js canary  | quisido/next.js patch-3 | Change |
| - | - | - | - |
| app-page-exp...dev.js gzip | 168 kB | 168 kB | ✓ |
| app-page-exp..prod.js gzip | 93.9 kB | 93.9 kB | ✓ |
| app-page-tur..prod.js gzip | 94.7 kB | 94.7 kB | ✓ |
| app-page-tur..prod.js gzip | 89.2 kB | 89.2 kB | ✓ |
| app-page.run...dev.js gzip | 138 kB | 138 kB | ✓ |
| app-page.run..prod.js gzip | 88.6 kB | 88.6 kB | ✓ |
| app-route-ex...dev.js gzip | 24 kB | 24 kB | ✓ |
| app-route-ex..prod.js gzip | 16.7 kB | 16.7 kB | ✓ |
| app-route-tu..prod.js gzip | 16.7 kB | 16.7 kB | ✓ |
| app-route-tu..prod.js gzip | 16.3 kB | 16.3 kB | ✓ |
| app-route.ru...dev.js gzip | 23.4 kB | 23.4 kB | ✓ |
| app-route.ru..prod.js gzip | 16.3 kB | 16.3 kB | ✓ |
| pages-api-tu..prod.js gzip | 9.37 kB | 9.37 kB | ✓ |
| pages-api.ru...dev.js gzip | 9.64 kB | 9.64 kB | ✓ |
| pages-api.ru..prod.js gzip | 9.37 kB | 9.37 kB | ✓ |
| pages-turbo...prod.js gzip | 21.9 kB | 21.9 kB | ✓ |
| pages.runtim...dev.js gzip | 22.6 kB | 22.6 kB | ✓ |
| pages.runtim..prod.js gzip | 21.9 kB | 21.9 kB | ✓ |
| server.runti..prod.js gzip | 49.4 kB | 49.4 kB | ✓ |
| Overall change | 930 kB | 930 kB | ✓ |

</details>

</details>
Commit: 80e34b2217b98c1a3fade3cdb8c2ca5a07c3987b

**Review by huozhi — APPROVED** [review]
Thanks!

**Pull request merged**